.gitignore

```diff
@@ -0,0 +1,7 @@
+/.phpcs-cache
+/.phpunit.result.cache
+/composer.lock
+/phpcs.xml
+/phpstan.neon
+/phpunit.xml
+/vendor/
```

Comment:
pls dvě skupiny, nejdřív složky pak soubory - je to na první pohled přehlednější